### PR TITLE
[CLOUDCRAFT-957] Diagram Multiple Cloud Accounts

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Document how to diagram multiple cloud accounts with Cloudcraft.

### Merge instructions
Please wait until the Cloudcraft team has reviewed the content before going ahead with the review and merging of the PR to avoid extra work.

- [x] Please merge after reviewing

### CODE REVIEW

**Review by licaroline — APPROVED** [review]

**Comment by hestonhoffman** [comment]
Opened a [docs ticket](https://datadoghq.atlassian.net/browse/DOCS-9195) for this one. Someone on the team will review soon!

**Review by srividya-rajagopalan — APPROVED** [review]

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Diagram Multiple Cloud Accounts
+---
+
+Cloudcraft is a tool designed to help visualize and plan cloud architecture in a seamless and efficient manner. While the current live experience does not support diagrams involving multiple cloud accounts, this guide explains how to achieve this using the legacy experience within Cloudcraft.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:5]:
```suggestion
Cloudcraft is a tool designed to help visualize and plan cloud architecture in a seamless and efficient manner. This guide explains how to use the legacy experience within Cloudcraft to diagram multiple cloud accounts. Consolidate multiple cloud accounts into a single cohesive diagram with the following steps.

```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Follow the steps below to consolidate multiple cloud accounts into a single cohesive diagram.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:7]:
```suggestion
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Multiple accounts diagram
+
+### 1. Enable the legacy experience
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:11]: If this is the only H2 level, recommend removing it and bumping child header out. Feels repetitive.
```suggestion
## 1. Enable the legacy experience
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+To work with multiple cloud accounts, use the legacy diagramming experience. Follow these steps to enable it:
+
+1. Open Cloudcraft and navigate to the **Live** tab.
+2. Locate the **New Live Experience** toggle switch and ensure it is turned off.
+
+{{< img src="cloudcraft/getting-started/diagram-multiple-cloud-accounts/new-live-experience-toggle.png" alt="Cloudcraft interface showing new live experience toggle for multiple cloud accounts." responsive="true" style="width:100%;">}}
+
+### 2. Layout the first account
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:20]:
```suggestion
## 2. Layout the first account
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Once the legacy experience is enabled, proceed with laying out the first account:
+
+1. Choose the initial cloud account you want to visualize.
+2. Start a scan of this account to gather its current architecture details.
+3. Select the **Auto Layout** button to automatically arrange the components of this account within the diagram.
+
+### 3. Layout the second account
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:28]:
```suggestion
## 3. Layout the second account
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+Cloudcraft is a tool designed to help visualize and plan cloud architecture in a seamless and efficient manner. While the current live experience does not support diagrams involving multiple cloud accounts, this guide explains how to achieve this using the legacy experience within Cloudcraft.
+
+Follow the steps below to consolidate multiple cloud accounts into a single cohesive diagram.
+
+## Multiple accounts diagram
+
+### 1. Enable the legacy experience
+
+To work with multiple cloud accounts, use the legacy diagramming experience. Follow these steps to enable it:
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:13]:
```suggestion
To work with multiple cloud accounts, you must enable the legacy diagramming experience:
```

Comment (estherk15) [reply content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:13]: Is "must" correct?

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Open Cloudcraft and navigate to the **Live** tab.
+2. Locate the **New Live Experience** toggle switch and ensure it is turned off.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:16]:
```suggestion
2. Locate the **New Live Experience** toggle and ensure it is switched **off**.
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+{{< img src="cloudcraft/getting-started/diagram-multiple-cloud-accounts/new-live-experience-toggle.png" alt="Cloudcraft interface showing new live experience toggle for multiple cloud accounts." responsive="true" style="width:100%;">}}
+
+### 2. Layout the first account
+
+Once the legacy experience is enabled, proceed with laying out the first account:
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:22]:
```suggestion
After the legacy experience is enabled, lay out the first account:
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Choose the initial cloud account you want to visualize.
+2. Start a scan of this account to gather its current architecture details.
+3. Select the **Auto Layout** button to automatically arrange the components of this account within the diagram.
+
+### 3. Layout the second account
+
+After successfully laying out the first account, you can incorporate additional accounts into the diagram:
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:30]:
```suggestion
After successfully laying out the first account, you can add more accounts into the diagram:
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Choose the second cloud account you wish to add.
+2. Perform a scan of the second account to capture its architecture.
+3. Select **Auto Layout** again. This time, access the **Options** dropdown and select **Include existing components.** This action ensures the components from the first account remain visible, integrating both accounts into a single diagram.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:34]:
```suggestion
3. Select **Auto Layout**. 
4. Access the **Options** dropdown and select **Include existing components.** This action ensures the components from the first account remain visible, integrating both accounts into a single diagram.
```

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+{{< img src="cloudcraft/getting-started/diagram-multiple-cloud-accounts/auto-layout-options.png" alt="Cloudcraft interface showing AWS inventory and live component options." responsive="true" style="width:100%;">}}
+
+<div class="alert alert-info">You can repeat this process to include more accounts in a diagram.</div>
+
+By following the above steps, you will have a consolidated view of multiple cloud accounts within one Cloudcraft diagram. Review the layout to ensure all necessary components are included and positioned as desired. Adjust any placements manually if needed to improve clarity or presentation.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:40]:
```suggestion
Follow the above steps to have a consolidated view of multiple cloud accounts within one Cloudcraft diagram. Review the layout to ensure all necessary components are included and positioned as desired. Adjust any placements manually if needed to improve clarity or presentation.
```

**Review by estherk15 — APPROVED** [review]
Looks good! Left some small suggestion, if they look ok, I can commit and merge for you!

File: content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md (hunk 6 above)
Comment (jamesponddotco) [reply content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md:13]: It is!

**Pull request merged**